<launch>
<remap from="odom" to="/pepper_robot/naoqi_driver/odom"/>
  <node pkg="octomap_server" type="octomap_server_node" name="octomap_server">
  
		<param name="resolution" value="0.05" />
		
		<!-- fixed map frame (set to 'map' if SLAM or localization running!) -->
		<param name="frame_id" type="string" value="odom" />
		<!-- maximum range to integrate (speedup!) -->
    <param name="sensor_model/max_range" value="5.0" />
    <param name="latch" value="false" />
    <param name="filter_ground" value="false" />
    <param name="pointcloud_max_z" value="1.5" />
    <remap from="cloud_in" to="/pepper_robot/camera/depth_registered/points"/>

  </node>

</launch>